source/docs/zero-to-robot/step-2/step-2-next-steps.rst

```diff
@@ -1,14 +1,18 @@
 Next Steps
 ==========
 
-Congratulations! You have completed step 2 and should now have a working software development environment! Step 3 of this tutorial covers updating the hardware so that you can program it, while Step 4 showcases programming a robot in the VS Code Integrated Development Environment (IDE). For further information you can read through the :ref:`VS Code section <docs/software/vscode-overview/index:VS Code Overview>` to familiarize yourself with the IDE.
+Congratulations! You have completed step 2 and should now have a working software development environment!
+
+Click :ref:`here <docs/software/softwareoverview/control-system-software:FRC Software Component Overview>` for an overview of all software components!
+
+Step 3 of this tutorial covers updating the hardware so that you can program it, while Step 4 showcases programming the robot and running a simple Benchtop Test Program.
 
 Specific articles that are advised to be read are:
 
 - :ref:`Visual Studio Code Basics <docs/software/vscode-overview/vscode-basics:Visual Studio Code Basics and the WPILib Extension>`
-- :ref:`WPILib Commands in Visual Studio Code <docs/software/vscode-overview/wpilib-commands-vscode:WPILib Commands in Visual Studio Code>`
+- `LabVIEW Basics <https://www.ni.com/getting-started/labview-basics/>`__
 - :ref:`Creating a Robot Program <docs/software/vscode-overview/creating-robot-program:Creating a Robot Program>`
-- :ref:`Building and Deploying Robot Code <docs/software/vscode-overview/deploying-robot-code:Building and Deploying Robot Code>`
+- :ref:`LabVIEW Tank Drive Tutorial<docs/software/labview/creating-robot-programs/tank-drive-tutorial:Tank Drive Tutorial>`
 - :ref:`Installing 3rd Party Libraries <docs/software/vscode-overview/3rd-party-libraries:3rd Party Libraries>`
```

Comment:
I would rather split this into LabVIEW articles to be read and Java/C++ articles to be read. We don't want LabVIEW teams reading VS Basics and vise versa.

Reply:
Okay it was all Java/C++ before and that was really biased.  Maybe I will do tabs and have LV on one tab and Java/C++ on the other?

Reply:
I used a tab to separate the examples